.github/workflows/node-task.yml

```
@@ -0,0 +1,24 @@
+name: Run Node.js Task
```

```suggestion
name: Run yarn script
```

Also wonder if we can add a `run-name` here to output which script we're running?

---

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name

---

I've tried this but not been able to see any evidence of it being used. 

Example run: https://github.com/alphagov/govuk-chat/actions/runs/9762483958/job/26946059073

It looks like it uses the name value from the job rather than the workflow file in a workflow call so this has no effect.